examples/getting_started/1-Introduction.ipynb

```diff
@@ -14,7 +14,7 @@
     "\n",
     "## Why HoloViews?\n",
     "\n",
-    "HoloViews is an [open-source](https://github.com/ioam/holoviews/) Python 2 and 3 library for data analysis and visualization. Python already has excellent tools like numpy, pandas, and xarray for data processing, and bokeh and matplotlib for plotting, so why yet another library?\n",
+    "HoloViews is an [open-source](https://github.com/holoviz/holoviews) Python 2 and 3 library for data analysis and visualization. Python already has excellent tools like numpy, pandas, and xarray for data processing, and bokeh and matplotlib for plotting, so why yet another library?\n",
```

Comment:
Wow! That is an ancient link. I just searched, and there are lots of others similarly outdated, so I opened a new issue listing those (https://github.com/holoviz/holoviews/issues/4685). Happy to fix this one, in the meantime.